### PR TITLE
Chemistry: Patch Balance

### PR DESCRIPTION
**Pull request by Fox-McCloud**
So, simply put, there's not really a point or reason to use pills over patches; they hold the same amount of chemicals and apply special reactions for a small subset of those chemicals...while still adding the reagents to directly to someone's bloodstream.

This nerfs and adjusts them a bit--the idea was inspired by Goon.

- Patches max volume is now 40 as opposed to the standard pill's 50.
- Patches transfer reagents to the person at half the efficiency of pills
 - This makes sense, from a realistic perspective, too; something you ingest is going to get a LOT more chemicals into you than something that transfers through the skin.

### CODE REVIEW

**Comment by taukausanake** [comment]
I was going to think the patch should be lower like 25. Then I read that the patch works half as good as a pill does. This seems like a good balance but what do I know about Chemistry? :P

File: code/modules/reagents/newchem/patch.dm [hunk 1]
```diff
@@ -5,9 +5,10 @@
 	icon_state = "bandaid"
 	item_state = "bandaid"
 	possible_transfer_amounts = null
-	volume = 50
+	volume = 40
 	apply_type = TOUCH
 	apply_method = "apply"
+	transfer_efficiency = 0.5 //patches aren't as effective at getting chemicals into the bloodstream.
```
Comment (Iamgoofball) [inline code/modules/reagents/newchem/patch.dm]: so why not just make it hold 20 volume and have maximum efficiency

**Comment by Fox-McCloud** [comment]
@Iamgoofball 

Simple; because that dramatically hurts use of where patches are meant to be used; for touch based chems, and that's not really necessary.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -32,7 +33,7 @@
 			if(reagents.total_volume)
 				reagents.reaction(M, apply_type)
 				spawn(0)
-					reagents.trans_to_ingest(M, reagents.total_volume)
+					reagents.trans_to_ingest(M, reagents.total_volume*transfer_efficiency)
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/pill.dm]: >trans_to_ingest

why did you port this dumb stuff from bs12

**Comment by Fox-McCloud** [comment]
I didn't. It's always been there.

**Comment by Robertha89** [comment]
so instead, you make it so there is no reason to use patches over pills? I can't really agree with this as a nerf, especially considering how harsh it is. Here is the thing, people still use pills, but patches are used more for healing, because nobody wanna be forced to eat a pill.
Reducing the container size would be fine, but really making it half as efficient will also nerf every single medkit, also, nobody will ever uses patches anymore because half the chems are not used, so why bother.
:-1: This is badly thought out.

**Comment by SamCroswell** [comment]
There is a reason to use patches over pills.
Touch-based reactions for things such as styptic, sulfadiazine, and synthflesh.
Y'know, the things it makes sense to use patches for.

**Comment by Fox-McCloud** [comment]
Not to mention all the various TOUCH based poisons out there or reagents that penetrate the skin and get an automatic bonus because of that.

Patches will still, by far, have their use, even with this change; a single styptic/silver sulfa patch is still going to heal a whopping 56 damage from a single patch (vs 68 currently). Most of a silver sulfa and styptic's benefits come from the up front TOUCH based reaction that happens---and that is completely and totally unchanged---it's still going to apply the full patch's volume worth of TOUCH based reactions.

**Pull request merged**